tbx/core/blocks.py

```diff
@@ -302,3 +302,72 @@ class StoryBlock(StreamBlock):
 
     class Meta:
         template = "patterns/molecules/streamfield/stream_block.html"
+
+
+class ServiceStoryBlock(StreamBlock):
+    h2 = CharBlock(
+        form_classname="title",
+        icon="title",
+        template="patterns/molecules/streamfield/blocks/heading2_block.html",
+    )
+    h3 = CharBlock(
+        form_classname="title",
+        icon="title",
+        template="patterns/molecules/streamfield/blocks/heading3_block.html",
+    )
+    h4 = CharBlock(
+        form_classname="title",
+        icon="title",
+        template="patterns/molecules/streamfield/blocks/heading4_block.html",
+    )
+    intro = RichTextBlock(
+        icon="pilcrow",
+        template="patterns/molecules/streamfield/blocks/intro_block.html",
+    )
+    paragraph = RichTextBlock(
+        icon="pilcrow",
+        template="patterns/molecules/streamfield/blocks/paragraph_block.html",
+    )
+    image = ImageBlock(
+        template="patterns/molecules/streamfield/blocks/image_block.html",
+    )
+    call_to_action = CallToActionBlock(
+        label="Call to Action",
+        template="patterns/molecules/streamfield/blocks/call_to_action.html",
+    )
+    contact_call_to_action = ContactCTABlock(
+        label="Contact Call to Action",
+        template="patterns/molecules/streamfield/blocks/contact_call_to_action.html",
+    )
+    pullquote = PullQuoteBlock(
+        template="patterns/molecules/streamfield/blocks/pullquote_block.html"
+    )
+    raw_html = RawHTMLBlock(
+        label="Raw HTML",
+        icon="code",
+        template="patterns/molecules/streamfield/blocks/raw_html_block.html",
+    )
+    mailchimp_form = RawHTMLBlock(
+        label="Mailchimp embedded form",
+        icon="code",
+        template="patterns/molecules/streamfield/blocks/mailchimp_form_block.html",
+    )
+    markdown = MarkdownBlock(
+        icon="code",
+        template="patterns/molecules/streamfield/blocks/markdown_block.html",
+    )
+    embed = EmbedBlock(
+        icon="code",
+        template="patterns/molecules/streamfield/blocks/embed_block.html",
+        group="Media",
+    )
+    showcase = ShowcaseBlock(
```

Comment:
As mentioned in stand-up, the showcase block needs to be removed from the StoryBlock streamfield.